#include "Exception.h"

ArrException::ArrException(const std::string& message) : domain_error(message) {};
